src/plugins/discover/public/application/main/utils/fetch_all.ts

```diff
@@ -165,6 +167,60 @@ export function fetchAll(
   }
 }
 
+export async function fetchMoreDocuments(
```

Comment:
This is another function that I think could benefit from a couple of unit tests.

Reply:
Added via https://github.com/elastic/kibana/pull/163784/commits/2837cf696f3045f3f6224404352b8e9ca3105f45